packages/pocket-ponics-app/App.js

```diff
@@ -15,14 +15,15 @@ import StartSeedlingsScreen from './screens/start-seedlings-screen'
 import ProfileScreen from './screens/profile-screen'
 import AuthLoadingScreen from './screens/auth-loading-screen'
 import LoginScreen from './screens/login-screen'
+import HarvestInstructionScreen from './screens/harvest-instruction-screen'
 
 import { createAppContainer, createSwitchNavigator } from 'react-navigation';
 import { createStackNavigator } from 'react-navigation-stack';
 
-const AuthStack = createSwitchNavigator({ 
-	AuthLoading: { screen: AuthLoadingScreen },
-	Login: { screen: LoginScreen } 
-})
+ const AuthStack = createSwitchNavigator({ 
+ 	AuthLoading: { screen: AuthLoadingScreen },
+ 	Login: { screen: LoginScreen } 
+ })
```

Comment:
Make sure there is not extra whitespace here - this would probably be caught by the linter